### PR TITLE
pkg/helm: use CreateNamespace and fix and improve errors and user output

### LINKED ISSUE #7: Add support for Helm's --create-namespace
So releases can be created also outside "system" namespaces. This should be helpful to https://github.com/kinvolk/lokomotive/pull/704.

### LINKED ISSUE #8: Use Helm's --wait option
To ensure, that installed components deploys correctly.

### PR DESCRIPTION
**Pull request by invidian**
See commit messages for details.

Closes #7

### CODE REVIEW

File: pkg/helm/util.go [hunk 1]
```diff
@@ -65,3 +67,11 @@ func loadValuesFile(path string) (map[string]interface{}, error) {
 
 	return values, nil
 }
+
+// log is a wrapper ovre UserOutput function to be passed to Helm client,
```
Comment (ipochi) [inline pkg/helm/util.go:71]: nit: over

Comment (invidian) [reply pkg/helm/util.go]: Oops, fixed. thanks!

**Comment by invidian** [comment]
@ipochi please have a look again, I rebased the PR.

**Comment by invidian** [comment]
Yes, it has run as part of https://github.com/kinvolk/lokomotive/pull/805 (where I used this PR + #11 for building the bootkube image).

**Pull request merged**